nucliadb/src/nucliadb/search/search/chat/ask.py

```diff
@@ -174,17 +179,7 @@ def _ndjson_encode(self, item: AskResponseItemType) -> str:
         return result_item.model_dump_json(exclude_unset=False, exclude_none=True) + "\n"
 
     async def _stream(self) -> AsyncGenerator[AskResponseItemType, None]:
-        # First stream out the find results
-        yield RetrievalAskResponseItem(results=self.main_results)
-
-        if len(self.prequeries_results) > 0:
-            item = PrequeriesAskResponseItem()
-            for index, (prequery, result) in enumerate(self.prequeries_results):
-                prequery_id = prequery.id or f"prequery_{index}"
-                item.results[prequery_id] = result
-            yield item
-
-        # Then stream out the predict answer
+        # First, stream out the predict answer
```

Comment:
According to @lferran and @ebrehault, the consumers of the stream *shouldn't* be relying on stream item ordering.

As we are not assuming any order on the predict stream, we'd have to wait until the reranking chunk comes. If that's after the text, we'll loose the "real time" feeling of the response. Otherwise, we could assume an order on the predict chunks creating a dependency between repos. I don't think any of those two options are worth if we can reorder chunks.